### PR TITLE
extended object name length

### PR DESCRIPTION
**Pull request by idovgalyov**
Provided length for object names is now 25 (instead of 8 in older versions).
It is made because some asteroids have names with more than 8 symbols.
Updated ReadMe's.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,11 @@ TO PROVIDE TRUE DOUBLE PRECISION CALCULATIONS. ALSO, THE INITIAL DATA
 FOR PLANETS IN "big.in" FILE WAS UPDATED (see http://ssd.jpl.nasa.gov).
 FOR ANY QUESTIONS ABOUT THIS MODIFICATION PLEASE CONTACT WITH ME USING
 THIS E-MAIL:
-idovgalyov@4xxi.com                                         2016-12-05
+idovgalyov@4xxi.com
```
Comment (smirik) [inline README.md]: There is no need to provide an e-mail in README. It is possible to create an issue directly on github.

File: README.md [hunk 2]
```diff
@@ -22,7 +22,11 @@ TO PROVIDE TRUE DOUBLE PRECISION CALCULATIONS. ALSO, THE INITIAL DATA
...
+                                                            2016-12-05
+NEW UPDATE: Provided length for object names is now 25 (instead of 8 in
```
Comment (smirik) [inline README.md]: No need to write "New update".
Simply:
[2016-12-05]: text of the update

File: mercury6.man [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
+
 -----------------------------------------------------------------------
 -----------------------------------------------------------------------
 THIS VERSION OF MERCURY6 PACKAGE WAS MODIFIED BY I. DOVGALEV
 TO PROVIDE TRUE DOUBLE PRECISION CALCULATIONS. ALSO, THE INITIAL DATA
 FOR PLANETS IN "big.in" FILE WAS UPDATED (see http://ssd.jpl.nasa.gov).
 FOR ANY QUESTIONS ABOUT THIS MODIFICATION PLEASE CONTACT WITH ME USING
 THIS E-MAIL:
-idovgalyov@4xxi.com                                         2016-12-05
+idovgalyov@4xxi.com
```
Comment (smirik) [inline mercury6.man]: See the comments above.

**Comment by idovgalyov** [comment]
I performed some tests to be sure that all is working.
Apollo and Apophis were used.

File: README.md (hunk 2 above)
Comment (idovgalyov) [inline README.md]: Ok

**Comment by ANtlord** [comment]
I have a problem with aei files. They contains different data from develop brach version. Look
```

                              A2

    Time (years)              a                     x                     y                     z
          0.0000000    2.7662429290698904   -2.2753621656718424    1.0629775544477247    0.4532455917053866
          1.0000000    2.7659770804962704   -1.2785676495236600   -2.4269273788779739    0.1621483752381384
          2.0000000    2.7647786901147509    2.0292533796863115   -2.1207916739116790   -0.4402391632539298
          3.0000000    2.7645303296007566    2.6295971749491889    1.1125834132671106   -0.4523077431457754
          4.0000000    2.7669740400066019   -0.4991595844261416    2.5645474477143724    0.1707554450201724
          5.0000000    2.7654849620108477   -2.5025219242863987   -0.5318372190927888    0.4460807960058559```

**Comment by ANtlord** [comment]
From develop aei contains something like that
```

                              A1

    Time (years)        long            M           a        e        i      peri     node       mass
          0.0000000  1.524813E+02  2.333514E+00  2.76624 0.077529  10.6155  71.8697  80.6116  0.000000E+00
          3.0000000  1.518762E+02  2.377865E+02  2.76453 0.078059  10.6181  71.2730  80.6032  0.000000E+00
          6.0000000  1.507323E+02  1.136720E+02  2.76534 0.077187  10.6094  70.1744  80.5579  0.000000E+00
          9.0000000  1.509219E+02  3.482060E+02  2.76576 0.076469  10.6093  70.3797  80.5422  0.000000E+00
         12.0000000  1.530671E+02  2.205242E+02  2.76835 0.075666  10.5912  72.7456  80.3215  0.000000E+00
         15.0000000  1.542360E+02  9.414609E+01  2.76632 0.076638  10.5988  73.9669  80.2691  0.000000E+00```

**Comment by idovgalyov** [comment]
@ANtlord 
Pushed the default setting for .aei output format. Thanks for note.

**Pull request merged**